platform/android/export/export.cpp

```diff
@@ -2274,6 +2278,79 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return have_plugins_changed || first_build;
 	}
 
+	Error get_command_line_file(const Ref<EditorExportPreset> &p_preset, const String &p_path, int p_flags, Vector<uint8_t> *command_line_file) {
+		bool use_32_bit_framebuffer = p_preset->get("graphics/32_bits_framebuffer");
+		bool immersive = p_preset->get("screen/immersive_mode");
+		bool debug_opengl = p_preset->get("screen/opengl_debug");
+		String cmdline = p_preset->get("command_line/extra_args");
+		int version_code = p_preset->get("version/code");
+		String version_name = p_preset->get("version/name");
+		String package_name = p_preset->get("package/unique_name");
+
+		Vector<String> command_line_strings = cmdline.strip_edges().split(" ");
+		for (int i = 0; i < command_line_strings.size(); i++) {
+			if (command_line_strings[i].strip_edges().length() == 0) {
+				command_line_strings.remove(i);
+				i--;
+			}
+		}
+
+		gen_export_flags(command_line_strings, p_flags);
+
+		bool apk_expansion = p_preset->get("apk_expansion/enable");
+		if (apk_expansion) {
+			String apk_file_name = "main." + itos(version_code) + "." + get_package_name(package_name) + ".obb";
+			String fullpath = p_path.get_base_dir().plus_file(apk_file_name);
+			String apk_expansion_public_key = p_preset->get("apk_expansion/public_key");
+			Error err = save_pack(p_preset, fullpath);
+
+			if (err != OK) {
+				EditorNode::add_io_error("Could not write expansion package file: " + apk_file_name);
+				return err;
+			}
+
+			command_line_strings.push_back("--use_apk_expansion");
+			command_line_strings.push_back("--apk_expansion_md5");
+			command_line_strings.push_back(FileAccess::get_md5(fullpath));
+			command_line_strings.push_back("--apk_expansion_key");
+			command_line_strings.push_back(apk_expansion_public_key.strip_edges());
+		}
+
+		int xr_mode_index = p_preset->get("xr_features/xr_mode");
+		if (xr_mode_index == 1 /* XRMode.OVR */) {
+			command_line_strings.push_back("--xr_mode_ovr");
+		} else {
+			// XRMode.REGULAR is the default.
+			command_line_strings.push_back("--xr_mode_regular");
+		}
+
+		if (use_32_bit_framebuffer)
```

Comment:
```
if (p_preset->get("graphics/32_bits_framebuffer")) {
  command_line_strings.push_back("--use_depth_32");
}
```

Reply:
This is not a great idea since declaring the variables specifies their type. The return type of the `get` method is not strictly a bool

Reply:
Gotcha. Thanks for moving them though.